app/clients/worker/celery_client.py

```
@@ -0,0 +1,34 @@
+            return func
+        return task_decorator
+
+    def run_task(self, task_name: str, *args, **kwargs):
+        return self.celery_app.send_task(task_name, args=args, kwargs=kwargs)
+
+
+
+
+# Example Usage
+
+from app.clients.worker.celery_client import CeleryClient
```

Ruff (0.8.2)

Redefinition of unused `CeleryClient` from line 4

(F811)